packages/accordions/src/elements/timeline/components/Content.tsx

```diff
@@ -6,16 +6,17 @@
  */
 
 import React, { forwardRef, HTMLAttributes } from 'react';
-import { StyledTimelineContent, StyledCircleStrokeIcon, StyledSeparator } from '../../../styled';
+import CircleIcon from '@zendeskgarden/svg-icons/src/12/circle-full-stroke.svg';
+import { StyledTimelineContent, StyledSeparator, StyledItemIcon } from '../../../styled';
 import { useTimelineItemContext } from '../../../utils';
 
 export const Content = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>((props, ref) => {
   const { icon, surfaceColor } = useTimelineItemContext();
 
   return (
     <>
-      <StyledSeparator surfaceColor={surfaceColor}>
-        {icon || <StyledCircleStrokeIcon />}
+      <StyledSeparator>
+        <StyledItemIcon surfaceColor={surfaceColor}>{icon || <CircleIcon />}</StyledItemIcon>
```

Comment:
All icons should receive the same styling treatment, whether custom or not. And `surfaceColor` should be targeted to the component for which it directly applies.